docker/Dockerfile

```diff
@@ -17,12 +17,17 @@ RUN apk add --no-cache curl tar xz unzip jq
 WORKDIR /opt
 
 # Install FSM
-RUN curl --location "https://github.com/mroote/factorio-server-manager/releases/download/${MANAGER_VERSION}/factorio-server-manager-linux-${MANAGER_VERSION}.zip" \
+RUN curl --location "https://github.com/OpenFactorioServerManager/factorio-server-manager/releases/download/$MANAGER_VERSION/factorio-server-manager-linux-$MANAGER_VERSION.zip" \
          --output /tmp/factorio-server-manager-linux_${MANAGER_VERSION}.zip \
     && unzip /tmp/factorio-server-manager-linux_${MANAGER_VERSION}.zip \
     && rm /tmp/factorio-server-manager-linux_${MANAGER_VERSION}.zip \
     && mv factorio-server-manager fsm
 
+# Setup Nginx
+    mkdir -p /run/nginx && \
+    chown nginx:root /var/lib/nginx
+
+
```

Comment:
remove nginx, no nginx is installed in the container

Reply:
done.